src/validation.cpp

```diff
@@ -505,13 +505,13 @@ class MemPoolAccept
 
     // Run the script checks using our policy flags. As this can be slow, we should
     // only invoke this on transactions that have otherwise passed policy checks.
-    bool PolicyScriptChecks(ATMPArgs& args, Workspace& ws, PrecomputedTransactionData& txdata) EXCLUSIVE_LOCKS_REQUIRED(cs_main);
+    bool PolicyScriptChecks(ATMPArgs& args, const Workspace& ws, PrecomputedTransactionData& txdata) EXCLUSIVE_LOCKS_REQUIRED(cs_main);
```

Comment:
Er, would it be alright if I changed the `Workspace` here back to non-const? I'd interpret Workspace as an "in-out" parameter used throughout `MemPoolAccept::AcceptSingleTransaction`. As I understand it, this PR makes it `const` for 2 of the intermediate steps based on some code analysis, but I think it might just be coincidentally unmodified inside those functions 😅 I'd like to be able to update the workspace inside `PolicyScriptChecks`/`ConsensusScriptChecks`.

Reply:
Sure, just remove the `const` again. While the changes here are correct (as in adding a `const` will still compile), they probably don't make sense conceptually.

Reply:
@glozow Absolutely! What is `const` is not necessarily const over time :)

Reply:
Coolio, thanks 🤗